app/code/Magento/Bundle/Model/Option/SaveAction.php

```diff
@@ -93,7 +105,7 @@ public function save(ProductInterface $bundleProduct, OptionInterface $option)
             }
         } else {
             if (!$existingOption->getOptionId()) {
-                throw new NoSuchEntityException(
+                throw new \Magento\Framework\Exception\NoSuchEntityException(
```

Comment:
Can we import this instead of having the full path? Just to align with other changes on the PR.